zipline/assets/roll_finder.py

```diff
@@ -226,3 +229,8 @@ def _active_contract(self, oc, front, back, dt):
             if back_vol > front_vol:
                 return back
         return front
+
+    def get_contract_center(self, root_symbol, dt, offset):
+        day = self.trading_calendar.day
+        rolls = self.get_rolls(root_symbol, dt, dt + (180 * day), offset)
+        return self.asset_finder.retrieve_asset(rolls[0][0])
```

Comment:
Just a style thing, what would you think of writing the final line as:
```python
sid, _ = rolls[0][0]
return self.asset_finder.retrieve_asset(sid)
```
I like explicitly unpacking the tuple, since it makes it clear what the value represents.